### DISCUSSION TITLE
Date .format behind by one day

### DISCUSSION THREAD

**Discussion by cdt-eth** [js-fns/date-fns#2741] [Q&A] [answered]
I have `"startDate": "2021-11-01"` & `"endDate": "2022-11-01"` and need them converted to `MM/dd/yyyy`, which I did (below), but it's showing the **_day before_**. I'm assuming it's some timezone issue, but how do I fix this or think about it? I'm in the USA and EST. Best way to account for this?

```
const start = format(new Date(startDate), 'MM/dd/yyyy');
const end = format(new Date(endDate), 'MM/dd/yyyy');

console.log(start);
console.log(end);

// 10/31/2021 but should be 11/01/2021
// 10/31/2022 but should be 11/01/2022
```

**Comment by fturmel** [comment, marked as answer]
Yeah, `date-fns` works with local time zones only at the moment. The problem is you're passing strings in the JS Date constructor which treats the input as UTC, but the `date-fns` functions work on local time zone representations of the date.

> [...] date-only strings (e.g. "1970-01-01") are treated as UTC, not local
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date#parameters

```js
import { format, parseISO } from "date-fns";

console.log(format(parseISO("2021-11-01"), "MM/dd/yyyy")); // 11/01/2021
console.log(format(new Date(2021, 10, 1), "MM/dd/yyyy"));  // 11/01/2021
```

**Reply by fturmel** [reply]
Also, if you're running this server-side in Node, another approach could be to set the environment's time zone to UTC explicitly with `process.env.TZ='UTC'`

**Reply by cdt-eth** [reply]
oh wow, perfect. Yeah `parseISO` is what I needed thank you!
